src/Tizen.NET.MaterialComponents/Components/MConfirmationDialog.cs

```diff
@@ -0,0 +1,233 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ElmSharp;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MConfirmationDialog : MDialog, IColorSchemeComponent
+    {
+        readonly int MaximumHeight = 500;
+        readonly int ItemHeight = 120;
```

Comment:
shouldn't we consider to other profile such as wearable?

Reply:
I added 'MaximumContentHeight' property, so the user can adjust height for wearable.

Reply:
I mean the default value, the default value should be adjusted respectively.